### ISSUE TITLE
ThemeProvider children type error

### ISSUE THREAD

**Issue by Tobertet** [yamankatby/react-native-material#56]
Whenever you wrap something with the ThemeProvider in order to customize the theme, Typescript complains about this:

> Property 'children' does not exist on type 'IntrinsicAttributes & ThemeProviderProps'

In order to solve it I had to go to ThemeProvider.d.ts and change line 34 from 
`export declare const ThemeProvider: React.FC<ThemeProviderProps>;`
to
`export declare const ThemeProvider: React.FC<PropsWithChildren<ThemeProviderProps>>;`

It looks like this is simply a TS typing issue. I would open a PR myself but unfortunately I don't have much time lately.

**Comment by ghmendonca** [comment]
This issue was [mentioned](https://github.com/yamankatby/react-native-material/issues/33) 8 months ago and still not fixed :/ looks like this library is not being maintened anymore

**Comment by kmefeu** [comment]
@ghmendonca @Tobertet, have you found a solution for that?

**Comment by ghmendonca** [comment]
@kmefeu no, I ended up adding this comment in the line above the provider:

`{/* @ts-expect-error https://github.com/yamankatby/react-native-material/issues/56*/}`

**Comment by kmefeu** [comment]
@ghmendonca  Thank you for your feedback
